assets/pages/bundles-landing/bundlesLandingReducers.js

```diff
@@ -107,6 +109,9 @@ function contributionReducer(
         error: parseContribution(action.amount.value, state.type).error,
       });
 
+    case 'SET_PAYPAL_ERROR':
```

Comment:
Can you please add test for this case in `bundlesLandingReducersTest.js`?

Reply:
Done.